lib/id3c/cli/command/etl/fhir.py

```diff
@@ -455,6 +456,8 @@ def process_encounter(db: DatabaseSession, encounter: Encounter,
     details: Dict[str, Any] = encounter_details({ **related_resources, **contained_resources })
     if patient_language:
         details['language'] = patient_language
+    if encounter_source:
+        details.update(encounter_source)
```

Comment:
Should this be under an overall `__provenance` key like we currently have in `sample.details`?

Reply:
I think that would be nice for consistency with `sample.details`.

Reply:
The key in `sample.details` uses only a single underscore, `_provenance`, as produced by the `id3c manifest parse…` commands.